### ISSUE TITLE
Usability suggestions

### ISSUE THREAD

**Issue by samanpwbb** [jdalt/twin-cities-aid-distribution-locations#17]
Hey, I decided to take note of a handful of usability improvements that could be made to the site:

- Font size is 10px on mobile in some areas, which doesn't meet minimum accessibility standards. We want our base font size to be about 15px
- Markers currently do not display the name of the place but this is a pretty straightforward adjustment.
- Popups do not scroll, which can be problematic for some of the popups with a lot of content on small screens.
- close and hamburger menu hit areas are too small on mobile (should be 40px minimum width/height).

I will have time to address these issues tonight unless someone gets there before me – happy to hear other suggestions as well :)

**Comment by jdalt** [comment]
I think improving the usability should be our number one near term priority. @samanpwbb I think you've pointed out the most egregious issues. The popup scrolling is really frustrating.

Here's a couple other things that @estherrose046, @comeoneileen, and I discuessed:
- The `X` used to close the popups is minuscule
- The outer padding is awful in the responsive view of smallest screens
- There's some gross margin/padding things going on in the responsive view of the legend
- Filters work on the sidebar cards but not on the map pins
- Some of the lighter legend colors (the yellowish one I think) can be difficult to see on some displays
- The flyTo effect on the sidebar seems to be broken or removed (haven't looked through all the code yet). If you're somewhere else on the map, it's not obvious that clicking something on the sidebar had any effect.

**Comment by samanpwbb** [comment]
Thanks for the additional points. I’m working on all these things in a branch now, pr soon :)

**Comment by JamesMGreene** [comment]
I'm not sure how folks are currently getting to this map in the first place but it would be awesome if a little intro text blurb could be displayed when you first land on the map to describe what the user is about to see if they were just given a direct link. 🤔

**Comment by jdalt** [comment]
Migrated to new repo
